README.plugin.md

```diff
@@ -7,6 +7,18 @@
 See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin)
 for how to install Foreman plugins
 
+## Migrating your plugin:
```

Comment:
So that documentation is available on the wiki currently, that's the best place to maintain it.